plugins/notion/server/notion.ts

```diff
@@ -37,7 +41,15 @@ const AccessTokenResponseSchema = z.object({
   bot_id: z.string(),
   workspace_id: z.string(),
   workspace_name: z.string().nullish(),
-  workspace_icon: z.string().url().nullish(),
+  workspace_icon: z
+    .string()
+    .nullish()
+    .transform((val) => {
+      if (val && (isUrl(val) || emojiRegexp.test(val))) {
```

Comment:
With this regex it's best to just call it inline rather than assign in module scope